src/middleware/socketio_middleware.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import io from 'socket.io-client';
+import humps from 'humps';
+import { startPresentation } from '../state/presentation/presentation.actionCreators';
+import { SET_ACTIVE_TEAM } from '../state/active_team/activeTeam.actionTypes';
+import type { Image } from '../models/Image';
+import type { Team } from '../models/Team';
+import type { Action } from '../state/Action';
+
+function socketioMiddleware(url: string) {
+  return (store: any) => {
+    const socket = io(url);
```

Comment:
Line 12 to 26 don't have to be in the return function because they don't use the state.